src/test/java/uk/nhs/prm/e2etests/performance/load/RoundRobinPoolTest.java

```diff
@@ -10,35 +10,35 @@
 
 class RoundRobinPoolTest {
 
-    private List<Integer> integers = of(1, 2, 3);
+    private final List<Integer> integers = of(1, 2, 3);
 
     @Test
-    public void shouldGiveFirstItemFirst() {
-        var pool = new RoundRobinPool<>(integers);
+    void shouldGiveFirstItemFirst() {
+        RoundRobinPool<Integer> pool = new RoundRobinPool<>(integers);
 
-        var first = pool.next();
+        Integer first = pool.next();
```

Comment:
Can we change this to it's primitive type?

```suggestion
        int first = pool.next();
```